check_and_sort.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+with open('.clabot') as f:
+    data = json.load(f)
+
+data['contributors'] = list(set(data['contributors']))
+data['contributors'].sort(key=lambda x: x.lower())
+
+with open('.clabot', 'w') as f:
+    json.dump(data, f, indent=2)
+    f.write("\n")
```

Comment:
Can we make this a pre-commit hook?

Reply:
Oooh, good idea!

Reply:
Done in https://github.com/conda/infra/pull/565/commits/0112893dafdd7b02d00298836de8aa15bdae49e6.